docs/integrations/apps/guide/best-practices.mdx

```diff
@@ -37,20 +37,23 @@ Rather than polling endpoints, get notified when updates occur by subscribing to
 
 ### Manage user session timeouts
 
-Add BigCommerce's JavaScript SDK to your single-click app's front-end to prevent users from getting logged out of the control panel while using your app. To do so, reference the following script in your app's client-side code:
-
-```js copy
-      https://cdn.bigcommerce.com/jssdk/bc-sdk.js
-```
-
-To perform some action when a logout occurs, specify an `onLogout` callback:
+The [BigCommerce SDK](https://www.npmjs.com/package/@bigcommerce/app-sdk-js) keeps apps in sync with the control panel. To consume the SDK, you should load it at the top of an application as a part of the main index.
```

Comment:
We could further clarify that this needs to be run as frontend code in the browser in order for it to work; people who use BE frameworks might appreciate this clarification